code/modules/tgui_input/say_modal/tgui_say_speech.dm

```diff
@@ -63,45 +40,21 @@
 			return TRUE
 	return FALSE
 
-/**
- * Force say handler.
- * Sends a message to the say modal to send its current value.
- */
-/datum/tgui_say/proc/force_say()
-	window.send_message("force")
-	stop_typing()
-
-/**
- * Makes the player force say what's in their current input box.
- */
-/mob/living/carbon/human/proc/force_say()
-	if(stat != CONSCIOUS || !client?.tgui_say?.window_open)
-		return FALSE
-	client.tgui_say.force_say()
-
 /**
  * Handles text entry and forced speech.
  *
  * Arguments:
- *  type - a string "entry" or "force" based on how this function is called
+ *  type - a string "entry" based on how this function is called
```

Comment:
You removed this var entirely, yeet the comment

Reply:
updated